comp/core/workloadmeta/mock/mock.go

```diff
@@ -18,6 +18,7 @@ type Mock interface {
 	wmdef.Component
 
 	// The following are for testing purposes and should maybe be revisited
+
```

Comment:
nit: unnecessary empty line

Reply:
It's needed because otherwise the linter thinks it's part of the function description and complains because the line needs to start with the function name.

Reply:
Oh Okay, thanks 👍